spec/gemstash/http_client_spec.rb

```diff
@@ -82,7 +82,7 @@
       context "with a client that specifies a user agent" do
         let(:http_client) do
           Gemstash::HTTPClient.new(faraday_client,
-            user_agent: "my-agent 6.6.6")
+                                   user_agent: "my-agent 6.6.6")
```

Comment:
You can improve the style of piece of code if you are modifying it for the purpose of the PR otherwise this is better suited for another PR 🙂

Reply:
I'll create an other PR for this :)